spec/controllers/api/api_stages_controller_spec.rb

```diff
@@ -15,36 +14,36 @@
   it 'creates stages' do
     guide = FactoryGirl.create(:guide, user: user)
     old_length = Stage.all.length
-    data = { stage: { name: Faker::Lorem.word,
-                      order: 0,
-                      soil: Faker::Lorem.words(2) },
-             guide_id: guide.id }
-    post 'create', data, format: :json
+    data = { attributes: { name: Faker::Lorem.word,
+                           order: 0,
+                           soil: Faker::Lorem.words(2) },
+             guide_id: guide.id.to_s }
+    post 'create', data: data, format: :json
     expect(response.status).to eq(201)
     new_length = Stage.all.length
-    expect(json['stage']['name']).to eq(data[:stage][:name])
-    expect(json['stage']['soil']).to eq(data[:stage][:soil])
+    expect(json['data']['attributes']['name']).to eq(data[:attributes][:name])
+    expect(json['data']['attributes']['soil']).to eq(data[:attributes][:soil])
     expect(new_length).to eq(old_length + 1)
   end
 
   it 'should return an error when wrong info is passed to create' do
-    # FIXME what is this spec testing? Maybe you should do some assertions on
+    # FIXME what is this spec testing? Maybe we should do some assertions on
     # the response message.
     data = {
       instructions: "#{Faker::Lorem.paragraph}",
-      guide_id: guide.id
+      guide_id: guide.id.to_s
     }
-    post 'create', data, format: :json
+    post 'create', data: data, format: :json
     expect(response.status).to eq(422)
   end
 
   it 'should return an error when a guide does not exist' do
-    data = { stage: { instructions: "#{Faker::Lorem.sentences(2)}",
-                      name: 'hello',
-                      order: 0 },
+    data = { attributes: { instructions: "#{Faker::Lorem.sentences(2)}",
+                           name: 'hello',
+                           order: 0 },
              guide_id: 1 }
-    post 'create', data, format: :json
-    expect(json['guide_id']).to eq("Could not find a guide with id 1.")
+    post 'create', data: data, format: :json
+    expect(json['errors'][0]['title']).to eq("Could not find a guide with id 1.")
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.